README.md

```diff
@@ -1,37 +1,45 @@
-# Steam Summer Minigame 2015 Auto-play Optimizer #
+# Steam Summer Monster Minigame Auto-script
```

Comment:
Need to leave the year in here somewhere so when people search for a script next year, it's easy to tell this isn't relevant to next year's game.